### ISSUE TITLE
Fix typos and hint text for video page lab

### ISSUE THREAD

**Issue by jdwilkin4** [freeCodeCamp/freeCodeCamp#56491]
We need to fix this typo here found in the final solution code 

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md

The typo should be fixed to say developers not `evelopers `

```html
    <h1>Front End Web Development</h1>
        <p>Front End Web Development involves designing and building user interfaces of websites using HTML, CSS, and
            JavaScript. It focuses on creating visually appealing, interactive, and user-friendly websites. Front End
            evelopers
            ensure accessibility, performance optimization, and
            adherence to web standards, blending creativity with technical expertise.</p>
        <section>
```

Also the wording in this hint text could be reworded

```md

Each `section` element should contain an `h2` element to give a title to that section as its first child.
```

let's reword it to the following:

```md
Each `section` element should start with an `h2` element that serves as the title for that section.
```

**Comment by jdwilkin4** [comment]
This has been opened for contribution. The first comprehensive PR created will be reviewed and merged. We typically do not assign issues to anyone other than long-time contributors.

If you would like to contribute and have not read the contributors docs, please do so here: https://contribute.freecodecamp.org/#/

If you have any issues with contributing, be sure to join us on the [contributors channel](https://discord.com/channels/692816967895220344/715074489422970962), or on the [contributors sub-forum](https://forum.freecodecamp.org/c/contributors/3)

## Files that need to be changed
https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md

Happy coding

**Comment by jericho1050** [comment]
working on this. 
But first, Imma read the contribution guidelines real quick.
